### DISCUSSION TITLE
Spark nlp 3.1.0 requirement in model hub

### DISCUSSION THREAD

**Discussion by SjoerdH** [JohnSnowLabs/spark-nlp#2965] [General]
In the model hub there already are a fair amount of models which require the spark nlp 3.1.0 library. However this version is (at this time) nowhere to be found online. When will it be available?

**Comment by maziyarpanahi** [comment]
There are 22 models out of 1500+ models for 3.1.0, this version is under development without any ETA for now. 
https://github.com/JohnSnowLabs/spark-nlp/pull/2907

**Reply by SjoerdH** [reply]
Thanks!
